tee-worker/identity/client-sdk/packages/client-sdk/docs/modules/request.md

```diff
@@ -104,13 +107,13 @@ is the signed payload.
 
 #### Defined in
 
-[lib/requests/call-ethereum.request.ts:31](https://github.com/litentry/client-sdk/blob/develop/lib/requests/call-ethereum.request.ts#L31)
+[lib/requests/call-ethereum.request.ts:32](https://github.com/litentry/client-sdk/blob/develop/lib/requests/call-ethereum.request.ts#L32)
```

Comment:
404 not found?

Reply:
All links are broken as they point to the old repository (They are auto-generated), we should create an issue to update the config and all the links